### PR TITLE
feat: support tier-ecp for ecp network policy

### PR DESCRIPTION
**Pull request by qingwusunny**

### CODE REVIEW

File: pkg/agent/datapath/policyBridge.go [hunk 1]
```diff
@@ -21,11 +21,13 @@ const (
 	EGRESS_TIER1_TABLE          = 20
 	EGRESS_TIER2_MONITOR_TABLE  = 24
 	EGRESS_TIER2_TABLE          = 25
+	EGRESS_TIER_ECP_TABLE       = 28
```
Comment (wu-cl) [inline pkg/agent/datapath/policyBridge.go:24]: 27 maybe better, 28 & 26 remain for further usage

File: pkg/agent/datapath/policyBridge.go [hunk 2]
```diff
@@ -21,11 +21,13 @@ const (
...
 	EGRESS_TIER3_MONITOR_TABLE  = 29
 	EGRESS_TIER3_TABLE          = 30
 	INGRESS_TIER1_TABLE         = 50
 	INGRESS_TIER2_MONITOR_TABLE = 54
 	INGRESS_TIER2_TABLE         = 55
+	INGRESS_TIER_ECP_TABLE      = 58
```
Comment (wu-cl) [inline pkg/agent/datapath/policyBridge.go:30]: ditto

File: pkg/agent/datapath/policyBridge.go (hunk 1 above)
Comment (qingwusunny) [reply pkg/agent/datapath/policyBridge.go:24]: maybe add a tier with monitor in future, then need 2 tables with connected serial numbers.
and tier-ecp only need to higer than tier2, so set 28

File: pkg/agent/datapath/policyBridge.go [hunk 3]
```diff
@@ -538,6 +562,8 @@ func (p *PolicyBridge) GetTierTable(direction uint8, tier uint8, mode string) (*
 			case POLICY_TIER3:
 				policyTable = p.egressTier3PolicyMonitorTable
 				nextTable = p.egressTier3PolicyTable
+			case POLICY_TIER_ECP:
+				return nil, nil, fmt.Errorf("monitor mode doesn't support tier-ecp")
```
Comment (wu-cl) [inline pkg/agent/datapath/policyBridge.go:566]: We should validate this restriction in webhook, or it will keep retry for 5 mins.

Comment (qingwusunny) [reply pkg/agent/datapath/policyBridge.go:566]: done

**Review by wu-cl — DISMISSED** [review]

**Review by wu-cl — APPROVED** [review]

**Review by echkenluo — APPROVED** [review]

**Pull request merged**